### DISCUSSION TITLE
v2.0.0-alpha.9

### DISCUSSION THREAD

**Discussion by johnfercher** [johnfercher/maroto#290] [Announcements]
<hr /><em>This discussion was created from the release <a href='https://github.com/johnfercher/maroto/releases/tag/v2.0.0-alpha.9'>v2.0.0-alpha.9</a>.</em>

**Comment by portal7** [comment]
Hi John!
Are there any new features or changes that could be described in this release?
Thanks in Advance

**Reply by johnfercher** [reply]
Hey @portal7, actually this is the main bundle of features available in v2. But I will add this information.
